frontend/views/containers/proposals/Mincome.vue

```diff
@@ -1,7 +1,7 @@
 <template lang='pug'>
   proposal-template(
-    title= 'L("Change minimum income")'
-    footer='L("According to your voting rules, 8 out of 10 members will have to agree with this.")'
+    title='Change minimum income'
+    footer='According to your voting rules, 8 out of 10 members will have to agree with this.'
```

Comment:
These are string literals, these must be surrounded in `L` functions as before.

Reply:
Hum, but if you look into `proposal-template.vue` you noticed the title and footer are i118n, so we are converting 2x the same thing. Also, it was breaking the layout. It would render literally `L"Change min....`

Reply:
That's because these would need to be bound via `:title=` and `:footer=`, per the Vue.js documentation, otherwise it's passed in as a string.

Both `i18n` and `L` functions take string literals only, and all user-facing string literals **must** be wrapped in one of them.

Reply:
But if I change it back to what it was, it's like this:

![image](https://user-images.githubusercontent.com/14869087/62742181-7a55ff00-ba35-11e9-8e83-73444402b8f2.png)


What should we do then?

Reply:
> That's because these would need to be bound via `:title=` and `:footer=`, per the Vue.js documentation, otherwise it's passed in as a string.

See again, Vue.js documentation: https://vuejs.org/v2/guide/components-props.html

Reply:
🤦‍♀ it feels well to be back to VueJS.